extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -102,6 +103,15 @@ export class InstrumentationService {
     vars = dict(),
     enableDataVars = true
   ) {
+    if (!target.isConnected) {
+      dev().error(
+        'ANALYTICS-INSTRUMENTATION',
```

Comment:
nit: create `const TAG = 'ANALYTICS-INSTRUMENTATION';`

Reply:
you got it!